src/main/java/seedu/partyplanet/ui/CommandBox.java

```diff
@@ -21,6 +24,11 @@
     private final CommandExecutor commandExecutor;
     private final AutoCompleter autoCompleter;
 
+    private final KeyCombination undo = new KeyCodeCombination(KeyCode.Z, KeyCombination.CONTROL_DOWN);
```

Comment:
I'm amazed that this works, because I don't actually see the overloaded method in [KeyCombination](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/input/KeyCombination.html#KeyCombination-javafx.scene.input.KeyCombination.Modifier...-)

Reply:
Scratch that, it's the [KeyCodeCombination](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/input/KeyCodeCombination.html) class, not `KeyCombination`